sys/include/crypto/ciphers.h

```diff
@@ -41,10 +53,10 @@ extern "C" {
  * threedes     needs 24  bytes                           <br>
  * aes          needs CIPHERS_MAX_KEY_SIZE bytes          <br>
  */
-#if defined(MODULE_CRYPTO_3DES)
-    #define CIPHER_MAX_CONTEXT_SIZE 24
-#elif defined(MODULE_CRYPTO_AES)
+#if defined(MODULE_CRYPTO_AES)
     #define CIPHER_MAX_CONTEXT_SIZE CIPHERS_MAX_KEY_SIZE
+#elif defined(MODULE_CRYPTO_3DES)
+    #define CIPHER_MAX_CONTEXT_SIZE 24
```

Comment:
I think 3DES has been removed already, right?

Reply:
Oh I didn't notice that. I will remove that define.

It is kinda confusing (atleast to me) that the [documentation](https://doc.riot-os.org/group__sys__crypto.html) still mentions that you can use `crypto_3des` by adding it to `USEMODULE`.

Reply:
Leftover from #6960